script/dev/start-server.sh

```diff
@@ -13,14 +13,6 @@ if [[ "$PWD" != "$HOME"* ]]; then
   exit 1;
 fi
 
-ORCH_URL_ROOT=${ORCH_URL_ROOT:-'https://firecloud-orchestration.dsde-dev.broadinstitute.org'}
+docker run --rm -it -v "$PWD":/working broadinstitute/dsde-toolbox render-templates.sh local $VAULT_TOKEN
```

Comment:
I do like cross-project consistency, but it's more important to me that the codebase is telling the story of exactly what it is doing today rather than making predictions about the future. I did not write render-templates (just dockerized what was already there). I don't want to add a `src/main` until we have `src/test`, and I don't want to predict we will have a `src/test` because 100% of the time I predict code will move in a certain direction it turns out to be wrong.

That said, I see the consistency argument, so I'm happy to leave this to your judgement.